src/Microsoft.Docs.Template/Models/attribute/SchemaContentTypeAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Docs.Build
+{
+    public enum SchemaContentType
```

Comment:
>SchemaContentType [](start = 16, length = 17)

I was using this enum in the prototype, but we can just subclass `SchemaContentType` and without have this enum.

Reply:
you mean check type of the attribute instead?

---
In reply to: [210817615](https://github.com/dotnet/docfx/pull/3209#discussion_r210817615) [](ancestors = 210817615)